### DISCUSSION TITLE
python 相關書籍分享

### DISCUSSION THREAD

**Discussion by wangtyphoon** [1101-GS4719-C/Python-Community-GS4719-C#693] [General]
1. 精通python 
對於第一次接觸python的人來說這並不是最淺顯易懂的書籍，但絕對能帶你初窺這項語言的門徑，第一部分主要是基礎語法的介紹;第二部分則是各大領域的應用，相形粗陋如果有要鑽研仍須購買進階書籍
2.網站擷取 使用python
這本書主要是介紹beautifulsoup這個套件，那個時候是因為想自動下載圖片，如果再蒐集資料的過程中想自動化，一定不要錯過能自己寫爬蟲的機會啦
3. 機器學習錦囊妙計 
從資料預處理，到圖像影像聲音個別方案，最終則接上如sklearn,keras,tensorflow等機器學習之套件，是一個完整的學習方案，但如果要往CNN鑽研可能需要更專門的書。
4.python 資料科學學習手冊 
活用pandas,numpy等資料處理套件，其實才是最適合不論是文理組的簡單應用，繪圖的部分從matplotlib,到seaborn應有盡有，最後則sklearn的應用實例
5.流暢的python
這本我到現在還是看不懂 ，也不知道從何著手，適合進階程式語言者。
以上書籍本人皆買過，歡迎使用過不同書籍的人交流經驗

**Comment by danwan1027** [comment]
要自學的人都應該要有一本工具書，感謝分享！

**Reply by Yunhans** [reply]
我覺得要三本

**Comment by williamyen0623** [comment]
謝謝分享，希望能少走一點冤望路

**Comment by manipulateed** [comment]
Python的運用非常廣泛，各式主題的書籍也很多，感謝分享～。

**Comment by dabbie109303501** [comment]
我自己在學習python時都沒有特地買過跟python有關的書籍，主要是覺得這些資料上網找都有，不過看完版主的分享，看書好像對自學python蠻有用的

p.s. 文章中第三點打成第二點ㄌ (應該是3. 機器學習錦囊妙計)

**Comment by Pietra1226** [comment]
學程式語言真的要工具書，隨時拿起來翻很有用

**Comment by CPCpyt** [comment]
感謝分享，我也有下單購買其中一本了。
希望能在寒假時精進自己。
